Examples/Swift/SwitchStylesExample.swift

```diff
@@ -16,21 +16,16 @@ class SwitchStylesExample: UIViewController, MGLMapViewDelegate {
         mapView.centerCoordinate = CLLocationCoordinate2D(latitude: 28.10, longitude: -81.76)
         mapView.setZoomLevel(5.4, animated: false)
         view.addSubview(mapView)
-        mapView.delegate = self
         
         let styleToggle = UISegmentedControl(items: ["Dark", "Streets", "Light"])
         styleToggle.translatesAutoresizingMaskIntoConstraints = false
         styleToggle.selectedSegmentIndex = 1
         view.insertSubview(styleToggle, aboveSubview: mapView)
         styleToggle.addTarget(self, action: #selector(changeStyle(sender:)), for: .valueChanged)
         
-        if #available(iOS 9.0, *) {
-            styleToggle.bottomAnchor.constraint(equalTo: mapView.logoView.topAnchor, constant: -20).isActive = true
-            styleToggle.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
-            styleToggle.widthAnchor.constraint(equalTo: view.widthAnchor, constant: -80).isActive = true
-        } else {
-            // Fallback on earlier versions
-        }
+        NSLayoutConstraint.activate(NSLayoutConstraint.constraints(withVisualFormat: "H:|-40-[styleToggle]-40-|", options: [], metrics: nil, views: ["styleToggle" : styleToggle]))
+        
+        NSLayoutConstraint.activate([NSLayoutConstraint(item: styleToggle, attribute: NSLayoutAttribute.bottom, relatedBy: NSLayoutRelation.equal, toItem: mapView.logoView, attribute: NSLayoutAttribute.top, multiplier: 1, constant: -20)])
```

Comment:
Nit: the type is inferred so we should be able to drop `NSLayoutAttribute` from `NSLayoutAttribute.bottom` (×3)